torch/csrc/utils/device_lazy_init.cpp

```diff
@@ -35,6 +35,10 @@ void device_lazy_init(at::DeviceType device_type) {
 
   std::string module_name = "torch." + at::DeviceTypeName(device_type, true);
   auto module = THPObjectPtr(PyImport_ImportModule(module_name.c_str()));
+  if (device_type == at::DeviceType::PrivateUse1 && !module) {
```

Comment:
In my opinion, the PrivateUse1 shoule have the same behavor as CUDA or XPU in lazy initializaion :D.
cc @albanD, take a look at this please.

Reply:
I would agree here. We might want to update the public doc about this, but I think at this point we require a device module to be registered for privateuse1 for a fully working install?

Reply:
Thanks for your comments, I will revert this change and update the doc in this PR.